### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by jana-scheidemann**
please review. thanks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <head>
+    <meta charset="UTF-8" />
+   
+    <title>Document title</title>
+  </head>
+
+  <body>
+    
+  <body>
+
```
Comment (HenrikBischkopf) [inline index.html:12]: add a footer here

**Review by HenrikBischkopf — COMMENTED** [review]
Looks good

File: shell-and-git/week1-fri.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+git switch -c <branchname>
+= create a new branch and switch to it
+
+git switch <branchname>
+=switch branches
+
+git branch
+=list your branches
+
+git branch -a =
```
Comment (olivernowotny) [inline shell-and-git/week1-fri.md:13]: Please remove the "=" sign

File: shell-and-git/week1-fri.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+list all branches (local and remote)
+
+git branch -d <branchname> =
```
Comment (olivernowotny) [inline shell-and-git/week1-fri.md:16]: Please remove the "=" sign

File: index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+<html lang="en-US">
+  <head>
+    <meta charset="UTF-8" />
+   
+    <title>Document title</title>
+  </head>
+
+  <body>
+    
+  <body>
```
Comment (HenrikBischkopf) [inline index.html:11]: closing tag here </body>

**Review by olivernowotny — CHANGES_REQUESTED** [review]
See comments

**Pull request merged**